### PR TITLE
Localization Improvment

### PR DESCRIPTION
**Pull request by urgiv**
add localization for: "Cooldown", "HotkeySet", "setBonus"
use Substitutions in localization
move "en-US" to folder "Localization"

### CODE REVIEW

File: Common/ArtificerItems.cs [hunk 1]
```diff
@@ -104,18 +106,18 @@ public override void ModifyTooltips(Item item, List<TooltipLine> tooltips)
...
                 coolTimeColorHex = "[c/be7878:";
                 tooltips.Insert(index, new TooltipLine(Mod, "cooldownTime",
-                "Cooldown time: " + $"{coolTimeColorHex}{finalCooldown} seconds] (" + $"Base: {baseCooldown} sec)"));
+                Language.GetTextValue("Mods.ArtificerMod.CommonItemtooltip.Cooldown.CooldowntimeColored").FormatWith(baseCooldown, finalCooldown)));
             }
             else
             {
                 tooltips.Insert(index, new TooltipLine(Mod, "cooldownTime",
-                "Cooldown time: " + $"{finalCooldown} seconds (" + $"Base: {baseCooldown} sec)"));
+                Language.GetTextValue("Mods.ArtificerMod.CommonItemtooltip.Cooldown.Cooldowntime").FormatWith(baseCooldown, finalCooldown)));
```
Comment (BlockCrusader) [inline Common/ArtificerItems.cs:120]: The tooltip for 'positive' cooldown times will need its own `CommonItemtooltip`, as it is colored differently from the negative tooltip;

- `[c/78be78:` is correct for the negative tooltips; as this colors the time text red.
- For the positive tooltip, we want `[c/be7878:`, which will color the time text green.

All you need to do is add an appropriate `CommonItemtooltip` to `localization` and use it here

**Review by BlockCrusader — CHANGES_REQUESTED** [review]
Thanks so much for this! It's a much needed improvement for the mod's localization, which was kinda a mess before since it made minimal use of the newer 1.4.4-gen systems. 
Only found one issue that needs fixing. Once that's done I'll get this merged and start working on publishing an update with this in it.

**Comment by urgiv** [comment]
should be good now, color code  [c/78be78] [c/be7878] confounded me

**Comment by BlockCrusader** [comment]
Alright, great! I'll merge this now. Thanks again!

The [c/78be78] and [c/be7878] things are hexadecimal codes. It's hard/impossible to use tML provided tooltip methods and such to only color part of a tooltip, so I used those to color just the text listing cooldown time.

**Review by BlockCrusader — APPROVED** [review]

**Comment by BlockCrusader** [comment]
Btw, I plan on getting these changes released to the Steam Workshop within the week. I wanna try adding a few new things and will also need to get those changes and some earlier ones pushed to this repository first so it's all unified for the next release.

**Pull request merged**